ndmg/graph/gen_graph.py

```diff
@@ -292,20 +292,9 @@ def save_graph(self, graphname, fmt="igraph"):
         elif fmt == "npy":
             np.save(graphname, nx.to_numpy_matrix(self.g))
         elif fmt == "igraph":
-            if self.modal == "dwi":
-                nx.write_weighted_edgelist(
-                    self.g, graphname, delimiter=" ", encoding="utf-8"
-                )
-            elif self.modal == "func":
-                np.savetxt(
-                    graphname,
-                    self.g,
-                    comments="",
-                    delimiter=",",
-                    header=",".join([str(n) for n in self.n_ids]),
-                )
-            else:
-                raise ValueError("Unsupported Modality.")
+            nx.write_weighted_edgelist(
+                self.g, graphname, delimiter=" ", encoding="utf-8"
```

Comment:
don't we want to make the edgelists comma delimited? (Based on an Issue you assigned to me that I haven't done yet)

Reply:
Yep! That'll likely be in a future branch - we're going to basically rewrite this function anyway, cause we don't care about having any other modalities.

Could change it now, but once we do that I'll have to rewrite some of `graphutils` to account for it, and make sure nothing else gets broken, so I want to do that formally in an independent branch